sdk/ai/azure-ai-projects/README.md

```diff
@@ -39,6 +39,232 @@ Use the returned token credential to authenticate the client:
 
 ## Examples
 
+### Agents
+
+The following steps outline the typical sequence for interacting with agents:
+
+1. Create a project client.
+2. Upload files and associate them with a vector store for file search.
+3. Create a thread.
+4. Create a message.
+5. Execute `Run`, `Run_and_Process`, or `Stream`.
+6. Clean up by deleting resources.
+
+Additionally, you can retrieve messages.
+
+#### Create Project Client
+
+To create a project client, use the following example:
+
+<!-- SNIPPET:sample_agents_basics.create_project_client -->
+
+```python
+project_client = AIProjectClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    conn_str=os.environ["PROJECT_CONNECTION_STRING"],
+)
+```
+
+<!-- END SNIPPET -->
+
+#### Create Vector Store and Upload File
+
+To enable agents to search your files, upload the files and associate them with a vector store:
+
+<!-- SNIPPET:sample_agents_file_search.upload_file_and_create_vector_store -->
+
+```python
+openai_file = project_client.agents.upload_file_and_poll(file_path="product_info_1.md", purpose="assistants")
```

Comment:
Why is this "openai_file"? why the "opeai"?

Reply:
Humm..   Let me call it "file" in sample.

Reply:
Fixed